node/as/http.js

```diff
@@ -228,7 +228,12 @@ TChannelHTTP.prototype.forwardToTChannel = function forwardToTChannel(tchannel,
                 var pair = head.headerPairs[i];
                 headers[pair[0]] = pair[1];
             }
-            hres.writeHead(head.statusCode, head.message, headers);
+            // work-around https://github.com/joyent/node/issues/25490
```

Comment:
may be a good idea to expand this url to a short comment explaining why the code is necessary